src/components/Articles.jsx

```diff
@@ -1,21 +1,43 @@
 import React, { useEffect,useState,useContext } from "react"
+import {useParams} from "react-router-dom"
 import axios from 'axios'
+import { getArticles } from "../utils/api"
 
 const Articles = () => {
+   
     const[articles, setArticles] = useState([])
-}
+
 
 useEffect(() => {
-    axios.get("")
-    .then(({res})=> {
-       return res.json();
-    })
-    .then((data) => {
-        setArticles(data.articles)
-    })
-})
+   getArticles().then((articles) => {
+   setArticles(articles)
+   })
+},[])
+
 
 
     return (
-        <div className="body">
-                    </div> )
\ No newline at end of file
+       <div>
+        <h1>Articles</h1>
+       {articles.map((article) => {
+        return (
+            <div key={article.article_id}> 
```

Comment:
This looks good! Could you please separate this "article card" into it's own component 🙂